regparser/grammar/terms.py

```diff
@@ -16,6 +16,7 @@
 
 e_tag = (
     Suppress(Regex(r"<E[^>]*>"))
+    + NotAny(Marker("this") + Marker("term")) 
```

Comment:
So, that modification is there because there's [a test for `<E T="03">This term</E> means should not match'`](https://github.com/cfpb/regulations-parser/blob/master/tests/layer_terms_tests.py#L128).

I really don't like the `NotAny()` there, so if there's a better way (or if that test is bunk), I'm happy to hear it. I was not able to find any similar `<E>This term</E> means` in any of the CFPB XML. I did however find `<P>This term means...</P>`, which won't match this anyway, but which suggests a remote possibility.